### ISSUE TITLE
Brake lights always on unless accelerating but only on multiplayer

### ISSUE THREAD

**Issue by wriley** [Mogli12/KeyboardSteer#127]
Version: 0.0.3.9
Problem: When playing multiplayer on a server the brake lights are always on unless one is pressing the accelerator
Expected Result: Brake lights only come on when braking and this is the behavior when playing single player

I did some debugging around this and found that some values are always zero during multiplayer, which is quite strange.

```
function keyboardSteerMogli:ksmUpdateWheelsPhysics( superFunc, dt, currentSpeed, acceleration, doHandbrake, stopAndGoBraking )
local brake = ( acceleration < 0 )
```

I printed acceleration and currentSpeed to screen and when playing single player saw the expected numbers, however when playing on a server they were both was always 0. Since acceleration is always zero the local "brake" variable is always false and other logic in this function makes the brake lights default to on.

There seems to be some kind of locality issue with client/server where some code is running on the server? I'm not sure what is going on or how the vehicle even moves if acceleration and currentSpeed are always 0.

I noticed the brake lights seem to behave normally with EnhancedVehicle mod and it looks like that mod is tracking local velocity and using that info for brake lights. I'm hoping someone smarter than me knows what's going on here.

**Comment by Mogli12** [comment]
Is it fixed with version 0.0.0.4 https://github.com/Mogli12/KeyboardSteer/commit/2ca7d889fa19799394b18721b3f8736b142b64a9?

**Comment by wriley** [comment]
Brake lights are working as intended with version 0.0.0.4 on multiplayer server, thank you!
